nearcore/src/config.rs

```diff
@@ -792,6 +792,41 @@ fn set_block_production_delay(chain_id: &str, fast: bool, config: &mut Config) {
     }
 }
 
+/// Type of the configuration to download.
+pub enum DownloadConfigType {
+    /// Validator node configuration.
+    Validator,
+    /// Non-validator RPC node configuration.
+    Rpc,
```

Comment:
nit (not a strong opinion): maybe better style would be all uppercase? we already have that style in the `DbKind` type: https://github.com/near/nearcore/blob/ea8ec289bc188bb6fecd0390a6409240b0f1590f/core/store/src/metadata.rs#L32 and